docs/guides/lists.md

```diff
@@ -0,0 +1,112 @@
+# Lists
+
+When you read or create resources, they get added to the `resources` object
+within your state slice. This is useful for quick lookups of a single
+resource, but there are a few things this doesn't let you do:
+
+1. keep track of resource ordering
+2. manage subsets of resources
+
+Lists allow you to do both of these things. We will consider these two use cases
+more in-depth in this guide, but first, let's cover how to use lists.
+
+### Using Lists
+
+To use a list, add the `list` property to your
+[CRUD operation actions](./crud-actions.md).
+
+```js
+import { actionTypes } fom 'resourceful-redux';
+import store from './store';
+
+store.dispatch({
+  type: actionTypes.READ_RESOURCES_PENDING,
+  resourceName: 'books',
+  list: 'mostPopular'
+});
+```
+
+You should use the same list for the lifetime of the request. For instance,
+if you use a particular list for the "pending" action, then you will want
+to use that same list for the "success" action, too.
+
+You only need to specify a list for create or read actions. Because lists
+only store IDs, they get resource updates automatically. And successful
+deletes will remove the deleted resources from all lists automatically.
```

Comment:
> And successful deletes will remove the deleted resources from all lists automatically.

gotta remove dis now or in my follow up PR

Reply:
This is the documentation for lists, so it should never be removed :)